test/build-model.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Response } from "express";
+import { buildModel } from "../src/index";
+import { MODEL_NOT_FOUND_ERROR } from "../src/constants";
+import { SequelizeQueryParserRequestInterface } from "../src/interfaces";
+
+const db = require("../example/db");
+describe("Build Model Middleware", () => {
+  test("Must throw an error if model param is not associate with an existing model and not call next", () => {
+    const middleware = buildModel(db);
+    const fakeNext = jest.fn();
+    expect(() =>
+      middleware({ params: { model: "pet" } } as any, {} as Response, fakeNext)
+    ).toThrow(MODEL_NOT_FOUND_ERROR);
+    expect(fakeNext.mock.calls).toHaveLength(0);
+  });
+
+  test("Must throw an error if modelName is not associate with an existing model and not call next", () => {
```

Comment:
Do not capitalize the test cases, also, you only corrected one, the rest still have the same original problem. Check the test cases in my branch for reference.